readme.md

```diff
@@ -80,17 +84,20 @@ Typescript 2 ASL is integrated with the `@ts2asl/asl-lib` module. This module ca
 * [jsonPath expressions](./examples/arrays.md#json-path-expressions)
 * deploytime configuration & integration with CDK
 
-As asl-lib implements the same behavior as AWS Step Functions would, it is possible to take a program and execute it locally (using `ts-node`) or write unit tests against that execute using `ts-jest` within having to deploy anything to AWS.
+As `asl-lib` implements the same behavior as AWS Step Functions, you can execute your TypeScript code locally (using `ts-node`) or write unit tests against that execute using `ts-jest` within having to deploy anything to AWS.
```

Comment:
@OlafConijn was this supposed to be asl-lib or t2asl?

Reply:
this is meant to be `asl-lib`, which is a library used by `ts2asl`